rules/_deprecated/threat_intel_filebeat7x.toml

```diff
@@ -1,12 +1,14 @@
 [metadata]
 creation_date = "2021/04/21"
-maturity = "production"
-updated_date = "2021/10/29"
+deprecation_date = "2022/01/12"
+maturity = "deprecated"
+updated_date = "2022/01/12"
 
```

Comment:
most of the changes were from the formatter. The primary changes are this and moving to the _deprecated path